docs/user-guide/security-features.md

```diff
@@ -1,156 +1,228 @@
+* Block always
+* Block if fix is available
+* Allow
+* Inherit
 
-When you select an environment, it automatically adopts the policy of the associated cluster. For example, if critical-level vulnerabilities are blocked globally but allowed in the `Cluster Security Policy`, the `Environment Security Policy` will inherit this allowance. Consequently, critical-level vulnerabilities will also be allowed in the `Environment Security Policy`.
+The Environment Security Policy inherits its settings from the Cluster Security Policy, following a hierarchical structure where each level inherits the policy from its upper level.
+
+When you select an environment, it automatically adopts the policy of the associated cluster. For example, if critical-level vulnerabilities are blocked globally but allowed in the Cluster Security Policy, the Environment Security Policy will inherit this allowance. Consequently, critical-level vulnerabilities will also be allowed in the Environment Security Policy.
 
 However, you have the flexibility to make explicit changes to the policy if needed. This empowers you to customize the policy to align with specific requirements or preferences.
 
-![](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/security-features/environment-security-policy.jpg)
+![Figure 11: Configuring Environment Security Policy](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/security-features/new/environment-security-policy-1.jpg)
+
+---
 
 ## Configure Application Security Policy
 
-The `Application Security Policy` operates on a similar principle as other policies. However, in the `Application Security Policy`, the policy is determined by both the Environment option and the Application option.
+The Application Security Policy operates on a similar principle as other policies and offers four options:
+
+* Block always
+* Block if fix is available
+* Allow
+* Inherit
+
+However, in the Application Security Policy, the policy is determined by both the Environment option and the Application option.
 
 When modifying the policy within a development environment, the changes will be applied to all applications within that specific development environment. This means that any adjustments made to the policy settings will be consistently applied across all applications associated with that particular development environment.
 
 This approach ensures uniformity and streamlined management of security policies within specific environments and their corresponding applications.
 
-![](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/security-features/application-security-policy.jpg)
+![Figure 12: Configuring Application Security Policy](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/security-features/new/application-security-policy-1.jpg)
 
-## Block or Allow Specific CVE Policies.
+---
 
```

Comment:
Can we also attach a screenshot showing the impact of these policies? i.e. We can show that an image is blocked for deployment if it contains a vulnerability that has been blocked as per policies. This will help the user visualize the result/impact of these policies.